PhysicsTools/IsolationAlgos/plugins/CITKPFIsolationSumProducerForPUPPI.cc

```diff
@@ -74,6 +75,7 @@ namespace citk {
         useValueMapForPUPPI = true;
       }
       else useValueMapForPUPPI = false;
+      if (c.exists("usePUPPINoLepton")) usePUPPINoLepton = c.getParameter<bool>("usePUPPINoLepton");
```

Comment:
::exists method is rather an exception left for cases for rather old and complex pre-7X software.
Define a ::fillDescriptions method https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp